frontend/src/components/VSearchResultsGrid/VAudioCollection.vue

```diff
@@ -64,23 +54,39 @@ export default defineComponent({
       type: String as PropType<ResultKind>,
       required: true,
     },
-    fetchState: {
-      type: Object as PropType<FetchState>,
-      required: true,
-    },
     /**
      * The label used for the list of audio for accessibility.
      */
     collectionLabel: {
       type: String,
       required: true,
     },
+    searchTerm: {
+      type: String,
+      required: true,
+    },
+    relatedTo: {
+      type: String as PropType<string | null>,
+      default: null,
+    },
   },
-  setup() {
-    const { isVisible: isSnackbarVisible } = useAudioSnackbar()
+  setup(props) {
+    const uiStore = useUiStore()
+
+    const audioTrackSize = computed(() => {
```

Comment:
```suggestion
		// Determine the size of the audio track based on the kind
		// of collection and the current breakpoint.
    const audioTrackSize = computed(() => {
```

Nit: but the nested ternaries are a lot to take in, maybe a comment could help.